web/src/containers/ApdApplication.js

```diff
@@ -71,6 +73,10 @@ class ApdApplication extends Component {
           <Sidebar place={place} />
           <div className="site-main p2 sm-p4 md-px0 ds-l-col--9">
             <TopBtns />
+            <h1 className="ds-h1 apd--title">
```

Comment:
I can't figure out what targets the `apd--title` class. If it's not used, let's take it out, otherwise, I need more coffee!